
From: James Courtier-Dutton <James@superbug.demon.co.uk>

Rui Sousa has been unreachable for a long time now, so I have taken over
the emu10k1 project on sf.net.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/MAINTAINERS |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN MAINTAINERS~emu10k1-maintainer-update MAINTAINERS
--- 25/MAINTAINERS~emu10k1-maintainer-update	Fri Aug  6 15:05:24 2004
+++ 25-akpm/MAINTAINERS	Fri Aug  6 15:05:24 2004
@@ -764,8 +764,8 @@ M:	saw@saw.sw.com.sg
 S:	Maintained
 
 EMU10K1 SOUND DRIVER
-P:	Rui Sousa	
-M:	rui.p.m.sousa@clix.pt	
+P:	James Courtier-Dutton
+M:	James@superbug.demon.co.uk
 L:	emu10k1-devel@lists.sourceforge.net
 W:	http://sourceforge.net/projects/emu10k1/
 S:	Maintained
_
